styleguide/components/61-layouts/general/info-layout/_info-layout.scss

```diff
@@ -14,5 +17,87 @@
       margin-right: -16.66%;
       margin-left: -16.66%;
     }
+
+    // Stub.
+    .image-gallery,
+    .video {
+      margin-bottom: 1.6rem;
+    }
+  }
+
+  .extra-info-box {
+    @include theme('border-color', 'color-primary--lighten-4', 'more-info-block-border-color');
+
+    display: flex;
+    flex-wrap: wrap;
+    width: span(12 of 12);
+    margin-top: 2.4rem;
+    padding-top: 4rem;
+    padding-bottom: 2rem;
+    border-top: 2px solid;
+    justify-self: center;
+
+    ul a {
+      @include theme('color', 'color-tertiary', 'extra-info-box-link-color');
+    }
+
+    &-image {
+      display: none;
+
+      @include desktop {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: span(4 of 12);
+      }
+
+      img {
+        width: 13rem;
+      }
+    }
+
+    &-content {
+      display: flex;
+      flex-wrap: wrap;
+      width: span(12 of 12);
+
+      @include desktop {
+        width: span(8 of 12);
+      }
+
+      > h3 {
+        width: 100%;
```

Comment:
flex: 1 0 100% or just flex-basis: 100%

width: 100% is confusing on block elements.

Reply:
Why is that confusing? :P I think flex: 1 0 100% is a lot more confusing then "width" :P

Reply:
Because block elements are 100% wide by default.
So here I am thinking you forgot how to HTML... only to figure out later that you had good reason to do so because the parent element is flex.

When you use flex-basis it's immediately clear for other developers what you are doing.

Reply:
👍